grid/recruitment/jobs/maps/jobs.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jest/no-export */
+
+import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
+
+export const jobsTest = (
+  provider: string,
+  options?: RecordingProcessOptions
+): void => {
+  describe(`recruitment/jobs/${provider}`, () => {
+    let superface: SuperfaceTest;
+
+    describe('ListJobs', () => {
+      beforeAll(() => {
+        superface = new SuperfaceTest({
+          profile: 'recruitment/jobs',
+          provider,
+          useCase: 'ListJobs',
+          testInstance: expect,
+        });
+      });
+
+      describe('when searching for published jobs', () => {
```

Comment:
suggestion: would be great to have pagination tested as well

Reply:
I will add the test in subsequent PR.